lib/handle-filters.js

```diff
@@ -62,6 +56,38 @@ function handleFilters(ast, filters, options) {
         return filterWithFallback(filter, filename, attrs, 'renderFile');
       }
     }
+
+    /**
+     * Get filter's options.
+     *
+     * @param node Filter node
+     * @param mergeGlobalOptions Merge local options with global, true by default
+     * @return {{}}
+     */
+    function getAttributes(node, mergeGlobalOptions) {
```

Comment:
Why are we adding an option for `mergeGlobalOptions`.  I don't think there's ever a time when we don't merge the global options.